### ISSUE TITLE
fresh clone fails 'make install' due to Utils.Path.parse

### ISSUE THREAD

**Issue by MarcelineVQ** [idris-lang/Idris2#194]
OS is arch linux.

After cloning a fresh idris2 repo and bootstrapping by following these steps:
```
git clone git@github.com:idris-lang/Idris2.git foo
cd foo
make bootstrap SCHEME=scheme
```
The bootstrapping completes. However when then typing `make install` there's a failure of
```
../../build/exec/idris2 --install prelude.ipkg
Uncaught error: INTERNAL ERROR: Can't change directory to /home/august/idris2-0.2.0
```
when library installation is attempted.

edit: cause of this is below

**Comment by MarcelineVQ** [comment]
Issue has disappeared as of 2005269c007ee7f5a75d0e17c3c4d552f4031524
The major change I tried was to manually delete my prefix dir of $HOME/.idris2
I don't see an obvious change in the commits that would have helped so the dir deletion is likely a factor, try that if you get this issue yourself and can't solve it with `make distclean` alone

**Comment by MarcelineVQ** [comment]
Gurp!
Well `make install` completes and I have a working `idris2`, however there's no libs in `$HOME/.idris2/lib/` or `$HOME/.idris2/idris2-0.2.0/lib/` aside from `libidris2_support.so`
Since idris2 loads libs like prelude and contrib just fine, it's likely that the libs that are being loaded are from the bootstrap in the git repo, e.g.
```
Installing /media/storage/home/august/projects/idrisprojects/foo/libs/prelude/build/ttc/Builtin.ttc to /home/august/projects/idrisprojects/foo/bootstrap/idris2-0.2.0/prelude
```
in the op.

**Comment by MarcelineVQ** [comment]
Actual cause of this is a bug in path separation. My $HOME ends with a trailing /, `/home/august/`, which is causing a problem due to two things, config.mk adds a / when it does `PREFIX ?= $(HOME)/.idris2` and the paths parser is either not consuming extra / or it's not dropping empty dirs created via multiple /

**Comment by MarcelineVQ** [comment]
In particular where parse claims:
```
||| - Repeated separators are ignored, so "a/b" and "a//b" both have "a" 
|||   and "b" as bodies.
```
This is untrue, given "a//b" only body `a` is created.

**Comment by MarcelineVQ** [comment]
resolved via #198
